todo_app.sql

```diff
@@ -0,0 +1,59 @@
+DROP DATABASE IF EXISTS todo_app;
+DROP USER IF EXISTS michael;
+CREATE ROLE michael WITH ENCRYPTED PASSWORD 'stonebreaker';
+CREATE DATABASE todo_app; 
+\connect todo_app;
+
+CREATE TABLE tasks (
+  id SERIAL PRIMARY KEY,
+  title varchar(255) NOT NULL,
+  description text,
+  created_at timestamp NOT NULL DEFAULT now(),
+  updated_at timestamp,
+  completed boolean NOT NULL DEFAULT FALSE
+);
+
+ALTER TABLE tasks DROP COLUMN completed,
+ADD COLUMN completed_at timestamp DEFAULT NULL, 
+ALTER COLUMN updated_at SET NOT NULL, 
+ALTER COLUMN updated_at SET DEFAULT NOW();
+
+INSERT INTO tasks
+(title, description)
+VALUES 
+('Study SQL', 'Complete this exercise'),
```

Comment:
This should be split into two different inserts, where the first one should also be setting your created_at and the your updated_at for that row to `now()`.